pkg/fixer/fixes/fmt.go

```diff
@@ -0,0 +1,29 @@
+package fixes
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/open-policy-agent/opa/format"
+)
+
+func Fmt(in []byte, opts *FmtOptions) (bool, []byte, error) {
```

Comment:
`Fmt` is the first and currently only implementation of a 'fix', next would be `:=` but I think that this interface is approx correct. where a fix has a byte slice input and some options where fix config can be set.

For example, if a single line or set of lines were to be fixed only, then that could be set in options if the fix supported it.

Reply:
Yeah, that makes sense to me! Just thinking out loud now but I suppose another option would be that the functions all returned `TextEdit` arrays just like the LSP client expects, but we would also implement a "remediator" in Regal that would take that an transform the input text using that. It's basically just a list of locations and changes, so I don't think it'll be hard to do... that way both the LSP and "native" commands would look the same, just passed onwards to different remediators.

But it was just an idea, I haven't considered if it's a good one or not, lol